tests/components/nina/test_config_flow.py

```diff
@@ -295,7 +296,7 @@ async def test_options_flow_entity_removal(hass: HomeAssistant) -> None:
     config_entry = MockConfigEntry(
         domain=DOMAIN,
         title="NINA",
-        data=DUMMY_DATA,
+        data=deepcopy(DUMMY_DATA) | {CONF_REGIONS: {"095760000000": "Aach"}},
```

Comment:
Not sure its needed here but doesn't hurt

Reply:
It is indeed needed, without that the test fails.